packages/assets-controllers/src/AssetsContractController.test.ts

```diff
@@ -56,31 +61,39 @@ const TEST_ACCOUNT_PUBLIC_ADDRESS =
  */
 async function setupAssetContractControllers({
   options,
-  useNetworkControllerProvider,
+  useNetworkControllerProvider = false,
```

Comment:
Previously, the controller was relying on `useNetworkControllerProvider` being evaluated as a falsy value when set to `undefined`.